server/utils/generateBoard.js

```diff
@@ -14,11 +14,11 @@ const generateBoard = async () => {
   });
 
   const board = await Board.create({
-    title: "New Board",
+    title: boardtitle,
     columns: [inProgressColumn, completedColumn],
   });
   
-  return [ board._id ]
+  return board._id
```

Comment:
As mentioned above, might make more sense to return the entire board model to reduce the DB queries we are doing in the controllers.

Reply:
Thanks for the reminder. What was originally intended for user creation was repurposed for general use without optimization in mind.